docs/1-install-and-setup.md

```diff
@@ -155,22 +155,11 @@ And that's it! If it's totally *borked*, file an issue and let us know.
 
 #### 1. Cannot find namespace 'firebase'.
 
-If you run into this error while trying to invoke `ng serve`, open `src/tsconfig.json` and add the "types" array as follows:
-
-```json
-{
-  "compilerOptions": {
-    ...
-    "typeRoots": [
-      "../node_modules/@types"
-    ],
-    
-    // ADD THIS
-    "types": [
-      "firebase"
-    ]
-  }
-}
+If you run into this error while trying to invoke `ng serve`, you need to update to the correct types with the following commands (from the root of a new project):
+
```

Comment:
@taylorjason2016 Not sure if it's a version problem. But i can confirm, that I did a fresh install on Windows 7 machine, trying to look into an issue and I didn't have to create any  directories as per your PR request. I may try doing an install on Ubuntu to confirm.